.github/workflows/deploy_image.yml

```
@@ -0,0 +1,48 @@
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+      - name: Log in to the Container registry
+        uses: docker/login-action
+        with:
+          registry: ${{ env.REGISTRY }}
+          username: ${{ github.actor }}
+          password: ${{ secrets.REPO_PASSWD }}
+      - name: Extract metadata (tags, labels) for Docker
+        id: meta
+        uses: docker/metadata-action@v3
```

actionlint (1.7.4)

the runner of "docker/metadata-action@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)